.github/workflows/build-package-docs.yaml

```
@@ -0,0 +1,95 @@
+name: Ansible package docs build
+on:
+  workflow_dispatch:
+    inputs:
+      github_fork:
+        description: 'GitHub Fork'
+        required: true
+        default: 'ansible'
+      github_branch:
+        description: 'GitHub Branch'
+        required: true
+        default: 'devel'
+      language:
+        type: choice
+        description: 'Language'
+        required: true
+        default: 'english'
+        options:
+        - 'english'
+        - 'japanese'
+      package_version:
```

The "package" word means a million things in different contexts. For example, it's an arbitrary folder with Python files.
```suggestion
      ansible-distribution-version:
```

---

could be "distribution package" too.

---

I take your point but, in this context, I think "Ansible package" is how we commonly refer to the community distribution that you get with `pip install ansible`.

For instance this release announcement: https://forum.ansible.com/t/release-announcement-ansible-community-package-9-3-0/4096

Or the PyPI page: https://pypi.org/project/ansible/

I think `ansible-package-version` is fine. "distribution" makes sense but maybe it's not quite as meaningful?

Also `ansible-community-package-version` might be the most complete thing we could use but it's pretty long and yamllint complains about it.

---

We can remove ambiguity in the description for the input too.